core/templates/pages/splash-page/splash-page.component.ts

```diff
@@ -15,111 +15,135 @@
 /**
  * @fileoverview Component for the Oppia splash page.
  */
+import { Component, OnInit } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
 
-require('base-components/base-content.directive.ts');
+import splashConstants from 'assets/constants';
+import { UrlInterpolationService } from
+  'domain/utilities/url-interpolation.service';
+import { SiteAnalyticsService } from 'services/site-analytics.service';
+import { WindowRef } from 'services/contextual/window-ref.service';
+import { WindowDimensionsService } from 'services/contextual/window-dimensions.service.ts';
+import { LoaderService } from 'services/loader.service.ts';
+import { UserService } from 'services/user.service';
 
-require('domain/utilities/url-interpolation.service.ts');
-require('services/contextual/window-dimensions.service.ts');
-require('services/site-analytics.service.ts');
-require('services/user.service.ts');
+export interface Testimonial {
+  quote: string,
+  studentDetails: string,
+  imageUrl: string,
+  imageUrlWebp: string,
+  borderPresent: boolean
+}
```

Comment:
Isn'ŧ this already defined in some other page?

Reply:
I have added a TODO( #11657 ) for testimonials to be  moved to a separate directive  at line number 81.